tests/components/hikvision/conftest.py

```diff
@@ -61,7 +61,6 @@ def mock_hikcamera() -> Generator[MagicMock]:
     with (
         patch(
             "homeassistant.components.hikvision.HikCamera",
-            autospec=True,
```

Comment:
why do we remove the autospec? It generally preshapes your mock so you don't have to prepare everything